aleth-vm/main.cpp

```diff
@@ -348,7 +348,7 @@ int main(int argc, char** argv)
         };
     }
     else if (mode == Mode::Trace)
-        onOp = st;
```

Comment:
@chfast I think assigning a function object to a `std::function` creates a copy of it inside `std::function`

But then in the end (line 388) we output the accumulated trace from the original object.

Reply:
Yes, your are right. I did not realized that `std::function` copies objects.